docs/tutorial/creating_your_first_admin_class/defining_entities.rst

```diff
@@ -256,6 +256,50 @@ Comment
         }
     }
 
+.. note::
+
+    For advanced usage, ``$id`` might be implemented as an object. The bundle will automatically resolve its string
+    representation from the ID object using ``$entity->getId()->__toString()`` (if implemented) when needed
+    (e.g., for generating url / rendering).
+
+    For example, in a use case where `InnoDB-optimised binary UUIDs`_ is implemented:
+
+    .. code-block:: php
+
+        <?php
+
+        class Comment
+        {
+            /**
+             * @var \Ramsey\Uuid\UuidInterface
+             * @Id
+             * @Column(type="uuid_binary_ordered_time", unique=true)
+             * @GeneratedValue(strategy="CUSTOM")
+             * @CustomIdGenerator(class="Ramsey\Uuid\Doctrine\UuidOrderedTimeGenerator")
+             */
+            protected $id;
```

Comment:
why protected?

Reply:
To be honest, I don't know. The original documentation uses protected. To follow the context, used the same way it does.

https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blame/3.x/docs/tutorial/creating_your_first_admin_class/defining_entities.rst#L220

Reply:
please use `private`, we should change the docs you mentioned